Foundation/NSXMLDTDNode.swift

```diff
@@ -62,7 +62,7 @@ public class NSXMLDTDNode : NSXMLNode {
         }
     } //primitive
     
-    public override init(kind: NSXMLNodeKind, options: Int) {
+    public override init(kind: NSXMLNodeKind, options: NSXMLNodeOptionRawType) {
```

Comment:
This seems like an API change. It is declared in both 32 bit and 64 bit Darwin targets as `public convenience init(kind: NSXMLNodeKind, options: Int)` why can't it be that here too?